sysutils/mackup/Portfile

```diff
@@ -21,9 +21,9 @@ long_description    Mackup backs ups your application settings in a safe \
 
 homepage            https://github.com/lra/mackup
 
-checksums           rmd160  cb669f52b1c7d85d4c4a819387087f8a0270b3c0 \
-                    sha256  052d8a46b918d8711fd50f103ad905403466feed6bd3d6676fa9805e8c13661b \
-                    size    70778
+checksums           rmd160  5a8d24a583b399c1e2293c679ac9e906305e4573 \
+                    sha256  7e86c3bc427f4ec4be56b23a225a9f1482fb2aeb917e4240f7fb9f54626f32d6 \
+                    size    72554
 
 python.default_version 310
```

Comment:
if possible, the default Python version should be updated to match MacPort's default version as set in the `python` PG. Currently that is 3.11 and it will be updated to the next release every year on January 1st.